applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.h

```diff
@@ -88,14 +103,22 @@ enum ble_hci_vs_led_function_mode {
 };
 
 /**
- * @brief Enable VREGRADIO.VREQH in NET core for getting +3dBm TX power
- *        Note, this will add +3 dBm for the primary advertisement channels
- *        as well even ble_hci_vsc_set_pri_ext_adv_max_tx_pwr() has been used
- * @param high_power_mode	Enable VREGRADIO.VREQH or not
+ * @brief Set the pins to be used for the FEM
```

Comment:
Picky: Should we say "Front End Module (FEM)" ?

Reply:
Changed to
```suggestion
 * @brief Set the pins to be used by the nRF21540 (Front End Module - FEM)
```